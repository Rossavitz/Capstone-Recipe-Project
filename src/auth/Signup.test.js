import React from "react";
import { render } from "@testing-library/react";
import { MemoryRouter } from "react-router";
import SignupForm from "./Signup";

it("matches snapshot", function () {
  const { asFragment } = render(
    <MemoryRouter>
      <SignupForm />
    </MemoryRouter>
  );
  expect(asFragment()).toMatchSnapshot();
});
