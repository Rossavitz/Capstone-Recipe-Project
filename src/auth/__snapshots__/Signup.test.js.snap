// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <form>
    <label
      for="firstName"
    >
      First Name:
    </label>
    <input
      id="firstName"
      name="firstName"
      placeholder="First Name"
      type="text"
      value=""
    />
    <label
      for="lastName"
    >
      Last Name:
    </label>
    <input
      id="lastName"
      name="lastName"
      placeholder="Last Name"
      type="text"
      value=""
    />
    <label
      for="username"
    >
      Username:
    </label>
    <input
      id="username"
      name="username"
      placeholder="Username"
      type="text"
      value=""
    />
    <label
      for="password"
    >
      Password:
    </label>
    <input
      id="password"
      name="password"
      placeholder="Password"
      type="text"
      value=""
    />
    <label
      for="email"
    >
      Email:
    </label>
    <input
      id="email"
      name="email"
      placeholder="Email"
      type="text"
      value=""
    />
    <button
      id="submit"
    >
      Submit
    </button>
  </form>
</DocumentFragment>
`;
